lib/parser/index.js

```diff
@@ -97,10 +105,9 @@ function parse(juttle, options) {
 
     options = processOptions(options, defaultResolver);
 
-    return parseModule({ source: juttle, name: 'main' })
+    return parseModule({ source: juttle, name: 'main', filename: (options.filename ? options.filename : 'main') })
         .then(function() {
             asts.main.modules = buildModuleDefs(asts);
-
```

Comment:
Unrelated change. (Moreover one I disagree with — in general I like to always have an empty line before a `return`.)